docs/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM graphcore/pytorch:latest
+
+# Need node14 to build doc HTML. Taken from https://askubuntu.com/a/1113339
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    gnupg
+RUN curl -sL https://deb.nodesource.com/setup_14.x  | bash -
+RUN apt-get -y install nodejs
+RUN npm install
+
+RUN git clone -b documentation https://github.com/huggingface/optimum-graphcore.git
```

Replace with the following before merging:

```suggestion
RUN git clone https://github.com/huggingface/optimum-graphcore.git
```